api/core/model_runtime/model_providers/openrouter/openrouter.yaml

```diff
@@ -73,3 +73,22 @@ model_credential_schema:
           value: llm
       default: "4096"
       type: text-input
+    - variable: vision_support
+      show_on:
+        - variable: __model_type
+          value: llm
+      label:
+        zh_Hans: Vision 支持
+        en_US: Vision Support
+      type: radio
```

Comment:
```
      type: radio
      required: false
      options:
        - value: 'true'
          label:
            en_US: 'Yes'
            zh_Hans: 是
        - value: 'false'
          label:
            en_US: 'No'
            zh_Hans: 否
```

Reply:
The rest should be like this.

Reply:
It won't be compatible with `openai_api_compatible` If modify the value to `true`, What to do next?
[        return super().get_customizable_model_schema(model, cred_with_endpoint)](https://github.com/langgenius/dify/blob/b9b28900b1dedf4b684dc875cc226695518354fc/api/core/model_runtime/model_providers/openrouter/llm/llm.py#L40)

Reply:
```
type: radio
required: false
options:
  - value: 'support'
    label:
      en_US: 'Yes'
      zh_Hans: 是
  - value: 'no_support'
    label:
      en_US: 'No'
      zh_Hans: 否
```